lib/Doctrine/DBAL/DBALException.php

```diff
@@ -28,21 +35,57 @@ class DBALException extends Exception
      */
     public static function driverExceptionDuringQuery(Driver $driver, Throwable $driverEx, $sql, array $params = [])
     {
-        $msg = "An exception occurred while executing '" . $sql . "'";
-        if ($params) {
-            $msg .= ' with params ' . self::formatParameters($params);
-        }
-        $msg .= ":\n\n" . $driverEx->getMessage();
+        $messageFormat = <<<MESSAGE
```

Comment:
Nowdoc is more appropriate if you pick the version w/o `\n`
```suggestion
        $messageFormat = <<<'MESSAGE'
```